javascript/operators/import_meta_resolve.json

```diff
@@ -0,0 +1,50 @@
+{
+  "javascript": {
+    "operators": {
+      "import_meta": {
+        "__compat": {
+          "description": "<code>import.meta.resolve</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/import.meta.resolve",
+          "spec_url": [
+            "https://html.spec.whatwg.org/multipage/webappapis.html#hostgetimportmetaproperties",
+            "https://gist.github.com/domenic/f2a0a9cb62d499bcc4d12aebd1c255ab"
```

Comment:
GitHub gists is not a recognized spec source. I believe the HTML spec is good enough?

Reply:
The spec alone sounds fine, although it's really dense. I think it would be great to link the explainer from https://github.com/mdn/content/pull/23882 at least, because it provides the best context for the feature and its decisions (particularly why it's synchronous), and is actually quite hard to find.

Reply:
The spec URL isn't appropriate for anything more than the spec. If you feel like the explainer is useful, feel free to add it to the content PR.